Stardew Valley Expanded/[CP] Stardew Valley Expanded/content.json

```diff
@@ -16321,6 +16321,7 @@
 {       //KROBUS 4 HEART EVENT
 	 "Action": "EditData",
 	 "Target": "data/events/forest",
+	 "When": { "DayOfWeek": "Monday, Tuesday, Wednesday, Thursday, Saturday, Sunday" },
```

Comment:
You can optionally negate the condition like this:
```js
"When": { "DayOfWeek |contains=Friday": false },
```

Reply:
Did you want me to change to the above and re-commit?

Reply:
Sure, I suggest changing it to match the existing code. That's just a suggestion though; FlashShifter is the one who'll review and merge the PR.

Reply:
Sounds good. I fixed it based on your recommendation. Thanks :)